### ISSUE TITLE
Added optional env vars for helper tool paths. This is to allow for usage of this tool within custom build rules. [phab:D1322]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1714]
_Created by **craigbarber** at 2017-02-09 18:30:53_:

**Comment by kythe** [comment]
_**craigbarber** wrote at 2017-02-09 18:32:14_:

  - fixed documentation header

**Comment by kythe** [comment]
_**craigbarber** wrote at 2017-02-09 18:33:12_:

  - fixed usage println

**Comment by kythe** [comment]
_**fromberger** commented inline (context lost during migration) at 2017-02-09 18:35:50_:

Minor nit: "to be utilized" could be dropped without loss of information, I think.

**Comment by kythe** [comment]
_**fromberger** commented inline (context lost during migration) at 2017-02-09 18:35:50_:

Alternatively, I think you could also say

```
: ${SHASUM_TOOL:=${PWD}/kythe/go/platform/tools/shasum_tool/shasum_tool}
: ${KINDEX_TOOL:=${PWD}/kythe/go/platform/tools/shasum_tool/shasum_tool}
```
though the `if`s are fine too.

**Comment by kythe** [comment]
_**fromberger** wrote at 2017-02-09 18:35:50_:

A couple suggestions; otherwise LGTM.

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2017-02-09 18:35:50_

**Comment by kythe** [comment]
_**craigbarber** wrote at 2017-02-09 19:21:54_:

  - incorporated CL feedback

**Comment by kythe** [comment]
_**craigbarber** closed via commit f461be2b4edff265c74f415c0fb3f338589723ef at 2017-02-09 20:44:19_
